type-definitions/index.d.ts

```diff
@@ -19,6 +19,8 @@ declare interface DisposeFn {
   (): void | Promise<any>;
 }
 
-export default function create<A>(f: (add: (a: A) => void,
+declare function create<A>(f: (add: (a: A) => void,
                                       end: (x?: A) => void,
                                       error: (e: Error) => void) => void | DisposeFn ): Stream<A>;
+
+export = create;
```

Comment:
Does importing it with the line `import create = require('@most/create')` fix this?
Example here: https://github.com/cyclejs/cyclejs/blob/master/most-adapter/src/index.ts#L12